packages/jest-runtime/src/writeFileAtomic.js

```diff
@@ -0,0 +1,219 @@
+/* eslint-disable */
+
+// THIS FILE IS COPIED FROM https://github.com/npm/write-file-atomic/blob/1597785cb9e1152056d905a357b3891b3295506d/index.js
+
+// Forked in order to support running in worker threads, where process.pid is shared with the main thread
+
+'use strict'
+module.exports = writeFile
+module.exports.sync = writeFileSync
+module.exports._getTmpname = getTmpname // for testing
+module.exports._cleanupOnExit = cleanupOnExit
+
+var fs = require('graceful-fs')
+var MurmurHash3 = require('imurmurhash')
+var onExit = require('signal-exit')
```

Comment:
these deps need to be in package.json, they're transitive now. 
btw, how about moving this file to some kind of `vendor/` directory?

Reply:
bah :(

Reply:
I can stick it in `vendor`